lib/tasks/maintenance.rake

```diff
@@ -0,0 +1,88 @@
+namespace :maintenance do
+  desc "Create full combination flow tests for the Coronavirus Find Support smart answer."
```

Comment:
The description is fine, but I think some comments would be good - even just an example of usage.

Reply:
This file is no longer required.